scripts/clean-cluster-kind.sh

```diff
@@ -4,10 +4,12 @@ set -x
 
 source scripts/common.sh
 
-IMAGE=$1
-if ${RUNTIME} exec -ti kind-control-plane crictl inspecti ${IMAGE} &> /dev/null ; then
-	${RUNTIME} exec -ti kind-control-plane crictl rmi ${IMAGE}
-fi
-
+for image in "$@"; do
+	if ${RUNTIME} exec -ti kind-control-plane crictl inspecti ${image} &> /dev/null ; then
+		echo "Delete image ${image}"
+		${RUNTIME} exec -ti kind-control-plane crictl rmi ${image}
+	fi
+done
 kubectl delete deploy cocl-operator -n confidential-clusters || true
+kubectl delete job compute-pcrs -n confidential-clusters || true
```

Comment:
Already left the comment in the other PR, but the job deletion should be handled by the operator